CHANGELOG.md

```diff
@@ -4,11 +4,16 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Added
+
+- Helm: support labels in values.yaml ([#201]).
```

Comment:
note: This PR number needs to be changed to `203`.

Reply:
Done